torch/nn/utils/rnn.py

```diff
@@ -61,37 +61,6 @@ class PackedSequence(PackedSequence_):
     #
     # See the note above in doc string (starting with ":attr:`data` can be on
     # arbitrary device...").
-
-    def __new__(cls, data, batch_sizes=None, sorted_indices=None, unsorted_indices=None):
```

Comment:
I see that you have edited construction of PackedSequence within our library, but doesn't removing this break BC for any  user code that uses this API?